pallets/evercity/src/tests.rs

```diff
@@ -482,6 +483,51 @@ fn it_token_burn_hasty() {
         ));
     })
 }
+// fuse
+
+#[test]
+fn fuse_is_blone() {
+    new_test_ext().execute_with(|| {
+        let fuse = Evercity::fuse();
+        assert_eq!(fuse, true);
+
+        assert_noop!(
+            Evercity::set_master(Origin::signed(2),),
+            RuntimeError::InvalidAction
+        );
+    })
+}
+
+#[test]
+fn fuse_is_intact_on_bare_storage() {
+    let mut ext: sp_io::TestExternalities = frame_system::GenesisConfig::default()
+        .build_storage::<TestRuntime>()
+        .unwrap()
+        .into();
+
+    ext.execute_with(|| {
+        assert_eq!(Evercity::fuse(), false);
+
+        assert_noop!(
+            Evercity::account_add_with_role_and_data(Origin::signed(1), 101, MASTER_ROLE_MASK, 0),
+            RuntimeError::AccountNotAuthorized
+        );
+        assert_ok!(Evercity::set_master(Origin::signed(1),));
+        // make amend
+        assert_ok!(Evercity::account_add_with_role_and_data(
+            Origin::signed(1),
+            101,
+            MASTER_ROLE_MASK,
+            0
+        ));
+
+        assert_eq!(Evercity::fuse(), true);
+        assert_noop!(
```

Comment:
(maybe) check also the case "set_master(Origin::signed(1),)" (repeatable set master also should not work when fuse == true, maybe account(1) was disabled by other masters and tries to restore himself). Now it's clear that it's now working, but maybe later we'll add additional logic